src/order/entities/order.ts

```diff
@@ -0,0 +1,21 @@
+import { OrderStatusEnum } from '../types/order-status-enum';
+import { PaymentStatusEnum } from '../types/payments-types-enum';
+
+export class Order {
+  id?: string;
+  status: OrderStatusEnum;
+  price: number;
+  payment: {
+    type: PaymentStatusEnum;
+    cardholder: string;
+    dueDate: Date;
+    value: number;
+  };
+  products: [
+    {
+      id: string;
+      price: number;
+      name: string;
+    },
```

Comment:
```suggestion
  products: ProductDto[],
```
```
export class ProductDto {
      id: string;
      price: number;
      name: string;
}
```
Please, put the correct decorators for each field